static/src/javascripts/__flow__/types/atoms.js

```diff
@@ -0,0 +1,45 @@
+// @flow
```

Comment:
shouldn't this be packaged in atom renderer?

Reply:
Nope, this is the native implementation of the services provided by the platform, it is key the only thing that frontend has to bring.

Reply:
oh sorry I confused two different things. Yes, totally, you're right!